deep/permissions.py

```diff
@@ -65,6 +65,46 @@ def has_permission(self, request, view):
         return projects_count == len(project_ids)
 
 
+class DeleteLeadPermission(permissions.BasePermission):
+    """Checks if user can delete lead(s)"""
+    def has_permission(self, request, view):
+        if request.method not in ('DELETE',):
+            return True
+
+        project_id = request.parser_context['kwargs']['project']
```

Comment:
@thenav56 Can you verify if this permission handler is okay?

Reply:
this seems okay. But just to be on the safe side, use `.get('project')`.

Reply:
However, I do not understand(or maybe if I forgot if it was there already) why we are passing project id. Shouldn't leads be sufficient to get project information?

Reply:
I think we are limiting a user's action to a specific project first.
We shouldn't be able to bulk delete leads of different projects. @thenav56 right?